plugins/Eq/EqCurve.cpp

```diff
@@ -65,6 +65,7 @@ QRectF EqHandle::boundingRect() const
 
 float EqHandle::freqToXPixel( float freq , int w )
 {
+	if (freq == 0) { return 0.0f; }
```

Comment:
```suggestion
	if (typeInfo<float>::isEqual(freq, 0.0f)) { return 0.0f; }
```
This might be slightly better for comparing two floats for equality.